.github/workflows/update-pr-spreadsheet.yml

```diff
@@ -0,0 +1,16 @@
+name: PR Event Workflow
```

Comment:
@GarvitSinghal47 Just a little detail, nothing blocking. if we could rename to "Update contributions spreadsheet" on this other related PRs, it'd help us when previewing all action runs. I can do it later as well.